app/route_rangers_api/views.py

```diff
@@ -127,32 +137,36 @@ def survey_p1(request, city: str):
 
 
 def survey_p2(request, city: str, user_id: str = None):
+    """
+    Question about trip
+    """
     print(request.method)
     user_id = request.session.get("uuid")
-    print(f'user_id:{request.session["uuid"]} - page 1')
-    if request.method == "POST":
-        survey_answer = SurveyAnswer.objects.get(user_id=user_id)
-        update_survey = RiderSurvey2(request.POST, instance=survey_answer)
-        update_survey.save()
-        return redirect(reverse("app:survey_p3", kwargs={"city": city}))
-    else:
-        form = RiderSurvey2()
+    route_id = request.session.get("route_id")
 
-    context = get_survey_context(city, form)
-
-    return render(request, "survey_p2.html", context)
-
-
-def survey_p2(request, city: str, user_id: str = None):
-    print(request.method)
-    user_id = request.session.get("uuid")
-    print(f'user_id:{request.session["uuid"]} - page 1')
+    print(f"user_id:{user_id} - page 2")
```

Comment:
Unless these (and line 160)are useful for checking stuff when testing, let's get rid of the print statements. I guess I'm ok if there is a logic to keeping these, if these were for testing something we already confirmed it worked, remove them

Reply:
deleted!